scripts/lib/lando-extras.js

```diff
@@ -0,0 +1,219 @@
+     * @returns {string} Prefixed path.
+     */
+    #getFullPath(file) {
+        return `${this.#ymlPath}${file}`;
+    }
+
+    /**
+     * Copy configuration template into local config.
+     *
+     * @param {Object} template Config template to enable
+     */
+    #enableTemplate(template) {
+        Object.keys(template).forEach((tKey) => {
+            if (!this.#localConfig[tKey]) {
+                this.#localConfig[tKey] = {};
+            }
+            Object.assign(this.#localConfig[tKey], template[tKey]);
+        });
+    }
+
+    /**
+     * Remove template configration keys from localConfig
+     *
+     * @param {Object} template Config template to remove
+     */
+    #disableTemplate(template) {
+        Object.entries(template).forEach(([tKey, tContent]) => {
+            Object.keys(tContent).forEach((key) => {
+                delete this.#localConfig[tKey][key];
+            });
+        });
+    }
+
+    /**
+     * Return true if a config template is enabled in localConfig
+     *
+     * @param {Object} config Configuration template to check.
+     * @returns {boolean}
+     */
+    #enabled(template) {
+        const NotEnabledException = {};
+        try {
+            Object.entries(template).forEach(([tKey, tKeyContent]) => {
+                Object.keys(tKeyContent).forEach((item) => {
+                    if (!this.#localConfig?.[tKey]?.[item]) {
+                        throw NotEnabledException;
+                    }
+                });
+            });
+        } catch (e) {
+            if (e === NotEnabledException) {
+                return false;
+            }
+            throw e;
+        }
+        return true;
+    }
+
+    /**
+     * Return an extra by name
+     *
+     * @param {string} name Name of extra to locate
+     * @returns {Object}
+     */
+    #get(extra) {
+        return this.#extras.find((e) => e.name == extra);
+    }
+
+    /**
+     * Enable all extras
+     */
+    enableAll() {
+        this.#extras.forEach((extra) => {
+            this.#enableTemplate(extra.template);
+            extra.enabled = true;
+        });
+    }
+
+    /**
+     * Disable All Extras
+     */
+    disableAll() {
+        this.#extras.forEach((extra) => {
+            this.#disableTemplate(extra.template);
+            extra.enabled = false;
+        });
+    }
+
+    /**
+     * Enable an extra by name.
+     *
+     * @param {string} name Name of extra to enable
+     */
+    disable(name) {
+        const extra = this.#get(name);
+        this.#disableTemplate(extra.template);
+        extra.enabled = false;
+    }
+
+    /**
+     * Enable an extra by name.
+     *
+     * @param {string} name Name of extra to disable
+     */
+    enable(name) {
+        const extra = this.#get(name);
+        this.#enableTemplate(extra.template);
+        extra.enabled = true;
+    }
+
+    /**
+     * Map extra configs.
+     *
+     * @param Callable callback
+     * @returns {any} Results of map call.
```

Comment:
```suggestion
     * @returns {any} callback Results of map call.
```

Reply:
This is an @returns so there isn't a name (and the function doesn't return the callback but rather the result of the user-supplied callback.